autogen/agentchat/contrib/capabilities/transform_messages.py

```diff
@@ -43,12 +43,14 @@ class TransformMessages:
         ```
     """
 
-    def __init__(self, *, transforms: List[MessageTransform] = []):
+    def __init__(self, *, transforms: List[MessageTransform] = [], to_print_stats: bool = True):
```

Comment:
we should rename `to_print_stats` to `verbose` to be more aligned with the rest of the codebase